src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -58,6 +68,13 @@ public Claim save(
 
         countyCourtJudgmentRule.assertCountyCourtJudgementCanBeRequested(claim, countyCourtJudgment.getCcjType());
 
+        Optional<Response> response = claim.getResponse();
+        if (response.isPresent()) {
+            if (claim.getState().equals(ClaimState.STAYED) && ResponseUtils.isAdmissionResponse(response.get())) {
+                claim = caseRepository.saveCaseEvent(authorisation, claim, LIFT_STAY);
+            }
+        }
```

Comment:
So now this can be simplified to
```
if (claim.getState().equals(ClaimState.STAYED)
            && claim.getResponse().filter(ResponseUtils::isAdmissionResponse).isPresent()) {
    claim = caseRepository.saveCaseEvent(authorisation, claim, LIFT_STAY);
}
```

Reply:
I like that, done.